docs/guide-diagrams.md

```diff
@@ -0,0 +1,21 @@
+Mermaid is a JS based diagramming tool that renders markdown style syntax to create/modify diagrams. 
+
+Mermaid has native support in Github.
+https://github.com/github/roadmap/issues/372
+
+Information about Mermaid and the types of diagrams available
+https://mermaid.js.org/intro/
+
+User guide for beginners
+https://mermaid.js.org/intro/n00b-gettingStarted.html
+
+Mermaid syntax cheat sheet 
+https://jojozhuang.github.io/tutorial/mermaid-cheat-sheet/
+
+Vscode extension Markdown Preview Mermaid Support
+https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid
+
+To view diagram in Vscode: right click on file and select ‘open preview’ after plugin has been installed (restart required)
```

Comment:
Can you add some more viewing/developing information from [this comment](https://github.com/redhat-appstudio/integration-service/pull/219#issue-1757381642) somewhere around here?